Automattic/jetpack
# My Jetpack: Bump it to version 1.0

Bumps y Jetpack version to 1.0

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* No changes, only version numbers
* Check the builds work
* Check if things still work
*
